_posts/2015-09-08-code-talks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "ePages talks REST API"
+date: "2015-09-08 08:04:00"
+icon: code
+tags: misc
+categories: tech-stories conferences api
+authors: ["Birgit"]
+---
+
+Have you signed up for the [`code.talks`](https://www.codetalks.de/) 2015 in Hamburg?
+Would you like to learn more about the challenges of documenting REST APIs?
+We will be happy to see you on our talk then.
+Interested in some key points?
```

Comment:
Besides Steffis comments: What is the headline of our talk? And it would be nice to know at what time our talk will take place and the duration of it